Dockerfile

```diff
@@ -1,14 +1,13 @@
 FROM alpine:3.13
 
-# Install packges needed
+# Install required packages
 RUN apk --no-cache add ca-certificates curl bash jq py3-pip && \
     pip install awscli
 
-COPY ecs-deploy /ecs-deploy
-RUN chmod a+x /ecs-deploy
+COPY ecs-deploy /usr/local/bin/ecs-deploy
+RUN ln -s /usr/local/bin/ecs-deploy /ecs-deploy
 
 COPY test.bats /test.bats
 COPY run-tests.sh /run-tests.sh
-RUN chmod a+x /run-tests.sh
 
-ENTRYPOINT ["/ecs-deploy"]
```

Comment:
I prefer this, because there is no confusion over the PATH used in the container.

Reply:
The reason for the change is that if you wrap the execution of the container with some initial execution of commands, then it feels un-natural to have to specify the path to this script at the root location. In our CI steps for example we read the git ref first from a file, execute a script to assume a IAM role, and then run the `ecs-deploy`. It would feel natural to have the script in the `PATH` to not provide this full path at the root level.